src/keylessBackup/KeylessBackupPhoneCodeInput.tsx

```diff
@@ -77,6 +126,33 @@ const styles = StyleSheet.create({
     color: colors.primary,
     ...typeScale.labelSemiBoldMedium,
   },
+  bottomSheetTitle: {
+    ...typeScale.titleSmall,
+    textAlign: 'center',
+  },
+  bottomSheetView: {
+    paddingHorizontal: 24,
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  bottomSheetBody: {
+    ...typeScale.bodyMedium,
+    textAlign: 'center',
+  },
+  buttonContainer: {
+    flexDirection: 'row',
+    marginTop: 37,
+    marginBottom: 9,
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: 96,
+  },
+  goBackButton: {
+    color: colors.primary,
+  },
+  skipButton: {
+    color: '#9CA4A9',
```

Comment:
Color pulled from the figma doc, doesn't seem to be any of the `colors` in the style type files.

Reply:
in the figma it says gray4 which is in the wallet, could we just use that? https://www.figma.com/file/Qj4kphntYzlYwex79XTcEj/Wallet-Recovery-1.5?type=design&node-id=4309-36620&mode=design&t=y05CTe2XxambYJtg-0

Reply:
in general if there is a font color in the designs that isn't in the design system I recommend following up with design about it instead of hardcoding in a new color

Reply:
I saw that, however gray4 in the wallet is #666666 so doesn't match the figma one

Reply:
updated to the correct color now